File: Creational/Lab3/Python/person.py
from abc import *

class Person(ABC):
    def __init__(self, id=0, name_and_surname="Jane Doe", phone="8 800 555 35 35", email="example@gmail.com"):
        self.id = id
        self.name_and_surname = name_and_surname
        self.phone = phone
        self.email = email

File: Creational/Lab3/C#/Person.cs
namespace C_;

public abstract class Person
{
    public int Id { get; set; }
    public string NameAndSurname { get; set; }
    public string Phone { get; set; }
    public string Email { get; set; }

    public Person(int id, string nameAndSurname, string phone, string email) { 
        Id = id;
        NameAndSurname = nameAndSurname;
        Phone = phone;
        Email = email;
    }

    public Person(){
        Id = 0;
        NameAndSurname = "Jane Doe";
        Phone = "8 800 555 35 35";
        Email = "example@gmail.com";
    }
}
